crates/ide-assists/src/handlers/auto_import.rs

```diff
@@ -723,7 +757,7 @@ fn main() {
             }
             ",
             r"
-            use test_mod::TestTrait;
+            use test_mod::TestTrait as _;
```

Comment:
We're still offering both options, right? What's causing us to pick the alias one in tests?